### ISSUE TITLE
RuntimeError: !!Loop Protection!! We have tried to solve 3 time(s) in a row.

### ISSUE THREAD

**Issue by tsuliwaensis** [Xonshiz/comic-dl#256]
Anyone else getting the error when trying to download from Readcomiconline.to? I have the lastest version of cfscrape.

```
RuntimeError: !!Loop Protection!! We have tried to solve 3 time(s) in a row.
```

**Comment by edolstra** [comment]
This appears to be fixed in a recent version of cloudscraper (1.2.30 should work).

**Comment by tsuliwaensis** [comment]
Thank you/.
